app-portage/repoman/repoman-2.3.0-r2.ebuild

```diff
@@ -21,13 +17,11 @@ KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~s
 SLOT="0"
 IUSE=""
 
-DEPEND="dev-lang/python-exec:2"
-
-RDEPEND="
-	dev-lang/python-exec:2
-	!<sys-apps/portage-2.3.0_rc
-	>=dev-python/lxml-3.6.0
-	"
+DEPEND="
+	!<sys-apps/portage-2.3.0_rc[${PYTHON_USEDEP}]
```

Comment:
`[${PYTHON_USEDEP}]` should not be used in blocker.
`app-portage/repoman` built with `PYTHON_TARGETS="python3_5"` would still collide with `sys-apps/portage-2.2.28` built with `PYTHON_TARGETS="python3_4"` (at least in `/usr/bin/repoman`).

Use `!<sys-apps/portage-2.3.0_rc`.